packages/contracts/tasks/rekey.ts

```diff
@@ -10,7 +10,7 @@ task('rekey', 'Generates a new set of keys for a test network').setAction(
       'deployer',
       'validator',
       'proxyAdminOwner',
-      'kromaBaseFeeRecipient',
+      'protocolFundRecipient',
```

Comment:
```suggestion
-kromaBaseFeeRecipient
+protocolVault
```

Reply:
ditto. not a vault but a withdrawal address.